File: domain/src/main/java/com/thinkzi/weather/domain/usecase/base/CompletableObserver.java
package com.thinkzi.weather.domain.usecase.base;

import io.reactivex.observers.DisposableCompletableObserver;

/**
 * provide a observer for the work result (complete or error)
 * */
public class CompletableObserver extends DisposableCompletableObserver {

    @Override
    public void onComplete() {

    }

    @Override
    public void onError(Throwable e) {

    }

}

File: domain/src/main/java/com/thinkzi/weather/domain/usecase/base/CompletableObserver.kt
package com.thinkzi.weather.domain.usecase.base

import io.reactivex.observers.DisposableCompletableObserver

/**
 * provide a observer for the work result (complete or error)
 */
class CompletableObserver : DisposableCompletableObserver() {

    override fun onComplete() {

    }

    override fun onError(e: Throwable) {

    }

}
